src/Components/Address/AddressAutocompleteInput.tsx

```diff
@@ -19,6 +19,10 @@ import {
   SelectedItemFromAddressAutoCompletion,
 } from "@artsy/cohesion"
 
+// NOTE: Due to the format of this key (a long string of numbers that cannot be parsed as json)
+// This key must be set in the env as a json string like SMARTY_EMBEDDED_KEY_JSON={ "key": "xxxxxxxxxxxxxxxxxx" }
+const { key: API_KEY } = getENV("SMARTY_EMBEDDED_KEY_JSON") || { key: "" }
```

Comment:
I have a vague memory of this being troublesome when it was outside the component but can't remember why.